certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "Application/Json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "AppToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "69a489d9-53fd-43aa-b04c-5c15ed4c73dd",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "UserToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "98ae7807-0902-4e66-9b0a-bfe0ecd204de",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "relationshipName": {
+                  "type": "string",
+                  "description": "relationshipName",
+                  "title": "Relationship Name",
+                  "x-ms-visibility": "advanced",
+                  "default": "Employee"
+                },
+                "relatedToCustomerRecordNumber": {
+                  "type": "string",
+                  "description": "relatedToCustomerRecordNumber",
+                  "title": "relatedToCustomerRecordNumber",
+                  "x-ms-visibility": "advanced"
+                },
+                "reciprocalRelationshipName": {
+                  "type": "string",
+                  "description": "reciprocalRelationshipName",
+                  "title": "Reciprocal Relationship Name",
+                  "x-ms-visibility": "advanced",
+                  "default": "Employer"
+                },
+                "isPrimary": {
+                  "type": "boolean",
+                  "description": "isPrimary",
+                  "title": "Is Primary?",
+                  "x-ms-visibility": "advanced",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "default": false
+                },
+                "startDate": {
+                  "type": "string",
+                  "description": "startDate",
+                  "title": "Start Date",
+                  "x-ms-visibility": "advanced"
+                },
+                "endDate": {
+                  "type": "string",
+                  "description": "endDate",
+                  "title": "End Date",
+                  "x-ms-visibility": "advanced"
+                }
+              },
+              "required": [
+                "reciprocalRelationshipName",
+                "relatedToCustomerRecordNumber",
+                "relationshipName"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/Organizations/{id}": {
+      "put": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.